packages/components/src/components/link/link.ts

```diff
@@ -9,7 +9,7 @@ import SolidElement from '../../internal/solid-element';
 /**
  * @summary A link component.
  * @documentation https://solid.union-investment.com/[storybook-link]/link
- * @status experimental
+ * @status stable
  * @since 1.2
```

Comment:
looking at our changelog, its stable since our init of the components => Verison 1.0.0